docs/lib/auth/fragments/android/check_authentication_state/10_checkSignInState.md

```diff
@@ -0,0 +1,146 @@
+<amplify-block-switcher>
+<amplify-block name="Java">
+
+```java
+Amplify.Auth.fetchAuthSession(
+    result -> {
+        AWSCognitoAuthSession cognitoAuthSession = (AWSCognitoAuthSession) result;
+            // Check refresh token is valid or not
+            switch (cognitoAuthSession.getUserPoolTokens().getType()) {
+                case SUCCESS:
+                    Log.i("AuthQuickStart", "Refresh token is valid.");
+                    break;
+                case FAILURE:
+                    Log.e("AuthQuickStart", "Refresh token expired. User must re-authenticate by signing in again.", cognitoAuthSession.getUserPoolTokens().getError());
+                    // Check whether AuthSession is expired
```

Comment:
this comment is confusing, i thought at this point in the code, the auth session is expired already